### PR TITLE
feat(3240): Event Landing View & Job List View - Visually differentiate virtual job from normal job

### PR DESCRIPTION
**Pull request by sagar1312**
## Context

Builds for virtual jobs are not executed in a event. In the UI, such builds are not easily distinguishable from non-virtual job builds.

## Objective

- Event landing view (V1 and V2)
  - Add a new icon “fast-forward“ for the virtual job in the workflow graph
image

- Job List View (V1 and V2)
  - Use the new icon “fast-forward“ for the virtual job under the first column representing the most recent build status
image

- Pipeline Options View
  - Display the label “Virtual“ (with gray background) next to the job name to indicate when the job is virtual
image

## References

https://github.com/screwdriver-cd/screwdriver/issues/3240

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
For the fast forward icon, should it be in a circle too for consistency?

File: tests/integration/components/pipeline-list-job-cell/component-test.js [hunk 1]
```diff
@@ -93,6 +157,26 @@ module('Integration | Component | pipeline list job cell', function (hooks) {
     assert.dom('.job-name').hasText('b');
   });
 
+  test('it renders a failed build for virtual job', async function (assert) {
```
Comment (tkyi) [inline tests/integration/components/pipeline-list-job-cell/component-test.js:160]: would this show up as a red fastforward icon?

Comment (sagar1312) [reply tests/integration/components/pipeline-list-job-cell/component-test.js:160]: yes

**Comment by sagar1312** [comment]
> For the fast forward icon, should it be in a circle too for consistency?

We don't have one with the circle border in the font library we are using https://fontawesome.com/v4/icons/

Also when the build is frozen, we use snowflake icon which is not enclosed with in a circle

**Comment by tkyi** [comment]
Hm in the workflowGraph it looks enclosed but it's not in the events list
Screenshot 2024-11-08 at 10 59 06 AM
Screenshot 2024-11-08 at 10 59 15 AM

**Comment by sagar1312** [comment]
> Hm in the workflowGraph in the main area and in the events list it looks enclosed, but it's not for the overall status in the events list Screenshot 2024-11-08 at 10 59 06 AM Screenshot 2024-11-08 at 10 59 15 AM

But, we don't have fast-forward icon inside a circle. For now, we can go with the one with out the circle border.

**Review by tkyi — APPROVED** [review]

**Pull request merged**